packages/x-ui-admin/src/react/components/RemoteSelect.tsx

```diff
@@ -1,17 +1,45 @@
 import * as React from "react";
 import { Constructor } from "@bluelibs/core";
-import { Collection, use, useData } from "@bluelibs/x-ui";
+import { Collection, QueryBodyType, use, useData } from "@bluelibs/x-ui";
 import { Alert, Select, SelectProps, Spin } from "antd";
 import { ObjectId } from "@bluelibs/ejson";
 
+export interface OnSelectDataChangeOptions {
+  onChange: () => void;
+  isArray: boolean;
+  value: any;
+}
+
 export type RemoteSelectProps = SelectProps<any> & {
   collectionClass: Constructor<Collection<any>>;
   field: string;
   idAsString?: boolean;
+  fetchFields?: QueryBodyType<any>;
```

Comment:
Let's use the `body` term as it is used everywhere for this please! You can also add a comment on top of it.